charms/knative-serving/config.yaml

```diff
@@ -37,3 +37,8 @@ options:
       use a default image.  For usage details, see 
       https://github.com/canonical/knative-operators/blob/main/charms/knative-serving/README.md#setting-custom-images-for-knative-serving.
     type: string
+  queue_sidecar_image:
```

Comment:
Could this be part of the existing `custom_image` configuration? IIUC, the `custom_image` will hold a yaml file that will then be passed as context for rendering the manifests, so perhaps we do not need a new config at all.

Reply:
I think we should separate these 2:
* the `custom_images` config populates the `spec.registry,override` field of the KnativeServing CR
* while the `queue-sidecar-image` is a specific field in the `spec.deployment` section
Using the same config for both and using the charm logic to split the `custom_images` config into 2 places of the CR can be confusing to users IMO
We can make it clear in the config description that which config affects which part of the CR, WDYT?

Reply:
Given that Daniela is OOO, I think what you mention makes sense and we should keep those two separate. Thus, I 'm resolving this.